ansible/roles/kubernetes/tasks/secrets.yml

```diff
@@ -81,15 +81,41 @@
 
 - include: gen_tokens.yml
   when: inventory_hostname == groups['masters'][0]
```

Comment:
Actually, these token files of `system:scheduler-{{ inventory_name }}.token` do nothing. If you check the script of `kube-gen-token.sh`, you will find that these files only use to check weather a token was already created. 
So even with skipped values, it works well. I just think taht it looks not so "perfect", because these files are different from master[0] to others ...